docs/views.rst

```diff
@@ -113,7 +113,7 @@ Method Based Dispatching
 
 For RESTful APIs it's especially helpful to execute a different function
 for each HTTP method.  With the :class:`flask.views.MethodView` you can
-easily do that.  Each HTTP method maps to a function with the same name
+easily do that.  Each HTTP method maps to a class method with the same name
```

Comment:
Actually, would this be clearer as “method of a class”, to reduce potential confusion with `classmethod` (since these are regular methods, not classmethods)?

Reply:
or even “corresponding method of a class”

Reply:
Yes, it would be better.

The "corresponding" seems redundant since we already have "...with the same name". I would prefer to use "maps to **a method of a class**", or "maps to **a method of the class**", "the class" here refers to the example class below.